### PR TITLE
OUT-648 | Filter drop-down shows companies even if companies are disabled

### PR DESCRIPTION
**Pull request by rrojan**
### Tasks

- [OUT-648 | Filter drop-down shows companies even if companies are disabled](https://linear.app/copilotplatforms/issue/OUT-648/filter-drop-down-shows-companies-even-if-companies-are-disabled)

### Changes

- [x] Filter out companies from backend if `Enable companies` has been turned off from workspace settings.

### Testing Criteria

- [x] [LOOM](https://www.loom.com/share/dae18f2e33b04eb39de8cc010ed1933c?sid=ec8567af-26d7-4e4c-b927-d3da82ae582e)

### Notes

- While testing this, please make sure to remember to turn *off* Enable companies from dashboard settings. Also remember to toggle it back on after testing.

### CODE REVIEW

**Review by aatbip — COMMENTED** [review]
lgtm. waiting for QA test before approval.

**Review by Chaewang12 — APPROVED** [review]
Verified

**Review by aatbip — CHANGES_REQUESTED** [review]
I think this changes should also be made for the `getClient` method of this class. It is used for the CU routes.

**Review by aatbip — APPROVED** [review]

**Pull request merged**
